README.md

```diff
@@ -1,23 +1,44 @@
-# purescript-http-methods
+# Http Methods
 
-[![Latest release](http://img.shields.io/github/release/purescript-contrib/purescript-http-methods.svg)](https://github.com/purescript-contrib/purescript-http-methods/releases)
-[![Build status](https://travis-ci.org/purescript-contrib/purescript-http-methods.svg?branch=master)](https://travis-ci.org/purescript-contrib/purescript-http-methods)
-[![Pursuit](http://pursuit.purescript.org/packages/purescript-http-methods/badge)](http://pursuit.purescript.org/packages/purescript-http-methods/)
-[![Maintainer: garyb](https://img.shields.io/badge/maintainer-garyb-lightgrey.svg)](http://github.com/garyb)
-[![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-lightgrey.svg)](http://github.com/thomashoneyman)
+[![CI](https://github.com/purescript-contrib/purescript-http-methods/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-http-methods/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](https://img.shields.io/github/release/purescript-contrib/purescript-http-methods.svg)](https://github.com/purescript-contrib/purescript-http-methods/releases)
+[![Pursuit](https://pursuit.purescript.org/packages/purescript-http-methods/badge)](https://pursuit.purescript.org/packages/purescript-http-methods)
+[![Maintainer: garyb](https://img.shields.io/badge/maintainer-garyb-teal.svg)](https://github.com/garyb)
+[![Maintainer: thomashoneyman](https://img.shields.io/badge/maintainer-thomashoneyman-teal.svg)](https://github.com/thomashoneyman)
 
 HTTP method type.
 
 ## Installation
 
+Install `http-methods` with [Spago](https://github.com/purescript/spago):
+
+```sh
+spago install http-methods
 ```
-bower install purescript-http-methods
-```
+
+## Quick start
+
+The quick start hasn't been written yet (contributions are welcome!). The quick start covers a common, minimal use case for the library, whereas longer examples and tutorials are kept in the [docs directory](./docs.)
```

Comment:
Maybe the quick start could be a simple demonstration of parsing / printing HTTP methods (for example: https://github.com/purescript-contrib/purescript-form-urlencoded#quick-start)

Reply:
Done.